src/RA_Integration.vcxproj

```diff
@@ -94,7 +94,7 @@
     <ProjectName>RA_Integration</ProjectName>
     <ProjectGuid>{825CB292-F069-4B27-82CF-3417746ACDF3}</ProjectGuid>
     <RootNamespace>RA_Integration</RootNamespace>
-    <WindowsTargetPlatformVersion>7.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```

Comment:
I'm gonna need to check, but I'm pretty sure it says 7.0 in VS. Yes it does, source tree says there are no changes either.